### PR TITLE
Update YC for Round of 16

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Round of 16 mappool
- Add Round of 32 match result

### CODE REVIEW

File: wiki/Tournaments/BTT/5/en.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ The **Beginner's Taiko Tournament 5** (***BTT5***) is a worldwide double-elimina
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag, 25€ |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag, 15€ |
 
+![](img/badge.png "BTT5 winner badge")
```
Comment (TPGPL) [inline wiki/Tournaments/BTT/5/en.md]:
```suggestion
![BTT5 winner badge](img/badge.png)
```

File: wiki/Tournaments/YC/1/en.md [hunk 2]
```diff
@@ -152,6 +183,39 @@ The Yuri Cup is run by various community members.
 
 ## Match results
 
+### Round of 32
+
+Thursday, 27 February 2020:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| MeAqua | 4 | 5 | 11:40 | [#1](https://osu.ppy.sh/community/matches/58760885) |
```
Comment (TPGPL) [inline wiki/Tournaments/YC/1/en.md]: no bold

File: wiki/Tournaments/BTT/5/en.md (hunk 1 above)
Comment (fajar13k) [reply wiki/Tournaments/BTT/5/en.md]: I think i got this style of writing from clayton. So maybe no changes?

Comment (cl8n) [reply wiki/Tournaments/BTT/5/en.md]: whatever you think looks better. on 4DM2 the point of my review was to remove duplicate alt/title text

**Pull request merged**